src/features/location/sagas.js

```diff
@@ -0,0 +1,20 @@
+import { call, put, takeLatest } from 'redux-saga/effects';
+import locationApi from '../../rest/locationRest';
+import LOCATION_TYPES from './types';
+import locationAction from './actions';
+
+function* fetchCurrentLocation() {
```

Comment:
@jomadoye I believe this actually fetches all the locations, not just the current location, so probably should be caleld "fetchLocations".